src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3969,7 +3968,7 @@ public partial interface IReadOnlyCollection<out T> : System.Collections.Generic
     {
         int Count { get; }
     }
-    public partial interface IReadOnlyDictionary<TKey, TValue> : System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>>, System.Collections.Generic.IReadOnlyCollection<System.Collections.Generic.KeyValuePair<TKey, TValue>>, System.Collections.IEnumerable
+    public partial interface IReadOnlyDictionary<TKey, TValue> : System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>>, System.Collections.Generic.IReadOnlyCollection<System.Collections.Generic.KeyValuePair<TKey, TValue>>, System.Collections.IEnumerable where TKey : object
```

Comment:
where did the object constraint come from?

Reply:
Nullable annotations. That is the only way to tell the compiler that TKey is non-nullable

Reply:
I could exclude it from this PR and then add them when adding nullable annotations. Makes sense since it is out of the scope of this PR.

Reply:
It's fine - I just hadn't pulled up to date so I wasn't seeing it on the interface